tests/proofs/liquidity-baking/lb-spec.md

```diff
@@ -71,10 +71,10 @@ We have one case for when `#ceildiv` results in an upwards rounding, and one for
         <senderaddr> Sender </senderaddr>
         <nonce> #Nonce(Nonce => Nonce +Int 2) </nonce>
         <knownaddrs> KnownAddresses </knownaddrs>
-        <paramtype> #Type(#LiquidityBakingParamType()) </paramtype>
+        <paramtype> %default |-> #Type(#LiquidityBakingParamType()) </paramtype>
```

Comment:
Can we initialize this map (for all entrypoints) in `#runProof`? That way we do not need to repeat ourselves for each claim.

Reply:
Oh, I notice only `%default` has been set up. Don't we need the other entry points too?

Reply:
Currently, we need a `%default` entrypoint in order to supply the proper semantics of the `SELF` instruction. However, I realized that all of the parameter type maps were initialized incorrectly leading to proof errors (this incorrect initialization also occurs in the Dexter contract so I will fix it there as well).